src/bin/inx-chronicle/migrations/mod.rs

```diff
@@ -9,30 +9,21 @@ use eyre::bail;
 
 pub mod migrate_20230202;
 
+/// The list of migrations, in order.
+///
+/// In order to add a new migration, change the `LATEST_VERSION` above and add an entry to this list
+/// in the form `migrate_YYYYMMDD`.
+
 pub const LATEST_VERSION: &str = migrate_20230202::Migrate::VERSION;
 
-macro_rules! migration {
-    ($($mig:expr),*$(,)?) => {
-        migration([$(Box::new($mig) as _),*])
-    };
-}
+const MIGRATIONS: &[&'static dyn DynMigration] = &[&migrate_20230202::Migrate];
```

Comment:
Only one minor comment: Can you put the entry into a newline so that it's more obvious that it's intended to be added to? You'll need to add a comment above the item to avoid formatting.